File: src/main/java/ch/itds/taglib/asset/pipeline/thymeleaf/expression/AssetExpression.java
package ch.itds.taglib.asset.pipeline.thymeleaf.expression;

import ch.itds.taglib.asset.pipeline.thymeleaf.asset.AssetResolver;

@SuppressWarnings("unused")
public class AssetExpression {

    public String path(String path) {
        return AssetResolver.getAssetPath(path);
    }
}

File: src/main/java/com/dra11y/assetThymeleafTaglib/expression/AssetExpression.kt
package com.dra11y.assetThymeleafTaglib.expression

import com.dra11y.assetThymeleafTaglib.asset.AssetResolver
import org.springframework.context.annotation.Import
import org.springframework.stereotype.Component

@Component
@Import(AssetResolver::class)
class AssetExpression(private val assetResolver: AssetResolver) {
    fun path(path: String?): String = assetResolver.getAssetPath(path!!)
}
